ui/packages/app/web/src/components/MetricsGraph.tsx

```diff
@@ -178,7 +178,7 @@ export const MetricsTooltip = ({
                       <tr>
                         <td className="w-1/4">At</td>
                         <td className="w-3/4">
-                          {moment(highlighted.timestamp).utc().format(timeFormat)}
+                          {format(new Date(highlighted.timestamp).getUTCDate(), timeFormat)}
```

Comment:
Should we move `format(new Date(date).getUTCDate()` as a utility function and use it everywhere, since it is repeated a lot? 

Maybe inside `shared/functions` package.